protostar/testing/cairo_cheatcodes/send_message_to_l2_cairo_cheatcode.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+from typing import Optional
+
+from protostar.starknet import RawAddress, Address, CairoOrPythonData
+from protostar.starknet.selector import Selector
+from protostar.testing.cairo_cheatcodes.cairo_cheatcode import CairoCheatcode
+
+
+class SendMessageToL2CairoCheatcode(CairoCheatcode):
+    @property
+    def name(self) -> str:
+        return "send_message_to_l2"
+
+    def build(self):
+        def send_message_to_l2(
+            function_name: str,
+            from_address: RawAddress,
+            to_address: RawAddress,
+            payload: Optional[CairoOrPythonData] = None,
+        ) -> None:
+            asyncio.run(
+                self.cheaters.contracts.send_message_to_l2(
+                    from_l1_address=Address.from_user_input(from_address),
+                    to_l2_address=Address.from_user_input(to_address),
+                    selector=Selector(function_name),
+                    payload=payload,
+                )
+            )
```

Comment:
Don't we want to catch and re-throw wrapped exceptions here?

Reply:
I am not sure if that's necessary. I think we should return validation errors instead of raising them.

cc @MaksymilianDemitraszek

Reply:
I opened thread on slack, we should catch those in `Cheatcode`

Reply:
Blocked by https://github.com/software-mansion/protostar/pull/1404